### ISSUE TITLE
Revise contributing.md

### ISSUE THREAD

**Issue by ethanstrominger** [hackforla/peopledepot#111]
- Remove unneeded section headings to have fewer levels.  
- Organize sections so related sections are together
- Address comments in pull request.  Below is record of a Slack discussion about these requests.

<img width="586" alt="image" src="https://user-images.githubusercontent.com/32078396/207451294-4d8a7ef1-87b3-4e3f-96e3-ecadc2d2067b.png">
<img width="321" alt="image" src="https://user-images.githubusercontent.com/32078396/207451786-85ff6ffd-9825-42bb-a700-d90d40627d66.png">
<img width="317" alt="image" src="https://user-images.githubusercontent.com/32078396/207451998-2bfa336e-e502-473d-b35e-a8a6e0ee6ebb.png">

**Comment by yoyoyojoe** [comment]
Hey Ethan, thanks for contribution! When you create an issue, can you follow the format of an Issue Template? They are intended to help put issues into context, make them easy to navigate, and pave the way for more shape and color on actionable next steps. In most cases, I think the Blank Issue option would work.

I see you have created a PR already. I'll leave some comments there and we can discuss further during our Thursday meeting.

**Comment by yoyoyojoe** [comment]
Thanks so much for your work. I tried to be thorough and went through most of the issues you listed and screenshot here, and I think most of them are addressed. Definitely may have missed a few, but I think we're making strides. More comments were added to PR #113: mostly small things such as consistency around formatting and capitalizing certain titles. I'm observing a pattern of capitalizing all heading levels (except "in", "and", "on", and notes in "(notes...)", etc, which I think is nice, but I'm also noticing that it's inconsistently applied. If it's going to live on, let's catch those minor things while our eyes are still in this file!
